cttportfolio.html

```diff
@@ -3,106 +3,171 @@
 <head>
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<meta charset="UTF-8">
-	<title>HTML Exercise 1</title>
+	<title>Christian Turner: Project</title>
 	<link rel="stylesheet" type="text/css" href="cttportfolio.css">
+	<link href="https://fonts.googleapis.com/css?family=Montserrat:700&display=swap" rel="stylesheet"> 
 </head>
 <body>
 	<nav>
-		<a href="cttportfolio.html">
-			<img src="https://placeholder.com/wp-content/uploads/2018/10/placeholder.com-logo3.png" alt="Logo replacement">
+		<a class="navlogo" href="cttportfolio.html">
+			<img src="https://i.imgur.com/G6PitD9.png" alt="Logo here">
+			<p>Placeholder</p>
 		</a> 
 		<div class="navlinks">
-			<a href="cttportfolio.html">Home</a>
-			<a href="google.com">About me</a>
-			<a href="google.com">Portolio</a>
-			<a href="google.com">Contact me</a>
+			<a href="#">Home</a>
+			<a href="#aboutme_jump">About me</a>
+			<a href="#portfolio_jump">Portolio</a>
+			<a href="#contactme_jump">Contact me</a>
 		</div>
 	</nav>
+	<header id="home_jump">
+		<video autoplay muted loop preload="metadata" id="frontpagevideo" poster="https://i.imgur.com/fsCfb5b.jpg">
+			<source src="backgroundvideo.mp4" type="video/mp4">
+		</video>
+		<div class="headertext">
+			<h1>Christian Turner</h1>
+			<h2>Full stack loser lmao</h2>	
+		</div>
+	</header>
 	<main>
-		<header>
-			<div class="headertext">
-				<h1>Page Intro Text Here</h1>
-				<h2>Page intro subtext here</h2>	
+		<div id="aboutme_jump" class="aboutme"><!--aboutme=am-->
+			<div class="am_main amcontainer">
+				<div class="am_main_img am_img">
+					<img src="https://images.unsplash.com/photo-1507003211169-0a1dd7228f2d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=634&q=80" alt="picture of me here">
+				</div>
+				<div class="am_text background_color">
+					<h4>About me</h4>
+					<p>About me text here About me text here About me text here About me text here About me text here About me text here About me text here About me text here About me text here </p>
+				</div>
 			</div>
-		</header>
-		<div class="aboutmediv">
-			<div class="aboutmemain">
-				<div class="aboutmemainchild">
-					<div class="aboutmeimg">
-						<img src="https://images.unsplash.com/photo-1507003211169-0a1dd7228f2d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=634&q=80" alt="picture of me here">
-					</div>
-					<div class="aboutmetext">
-						<h4>about me</h4>
-						<p>About me text here About me text here About me text here About me text here About me text here About me text here About me text here About me text here About me text here </p>
+			<button type="button">View more</button>
```

Comment:
Is this meant to do nothing??

Reply:
yeah i plan on having the button expand the subpages for the about me part and portfolio part but idk how to do that yet so for now its like that.